guacamole/src/main/webapp/app/navigation/directives/guacUserMenu.js

```diff
@@ -164,6 +191,40 @@ angular.module('navigation').directive('guacUserMenu', [function guacUserMenu()
              */
             $scope.actions = [ LOGOUT_ACTION ];
 
+            // Check if Internet Explorer UserAgent
+            var ua = window.navigator.userAgent;
+            var isIE = (ua.indexOf('MSIE ') > -1) || (ua.indexOf('Trident/') > -1);
```

Comment:
> ... don't show toggle is running Internet explorer as it renders a black screen (known bug, but it's IE..)

I'm wary of browser sniffing like this unless absolutely necessary. @echu2013, can you point to where the bug is documented?

Reply:
I found several messages on forums regarding black screen with Internet Explorer but no official documentation by Microsoft of the issue.

@mike-jumper would you prefer the latest commit to be removed ?

Reply:
Sure, that would be OK with me.

Reply:
@mike-jumper done. Added a commit to revert the latest modification.